lib/em-http/http_connection.rb

```diff
@@ -94,6 +94,16 @@ def ssl_verify_peer(cert_string)
       end
 
       def ssl_handshake_completed
+        # Warning message updated by Ably — the previous message suggested that
+        # when verify_peer is false, the server certificate would be verified
+        # but not checked against the hostname. This is not true — when
+        # verify_peer is false, the server certificate is not verified at all.
+        unless verify_peer?
+          warn "[WARNING; ably-em-http-request] TLS server certificate validation is disabled (use 'tls: {verify_peer: true}'), see" +
+               " CVE-2020-13482 and https://github.com/igrigorik/em-http-request/issues/339 for details" unless parent.connopts.tls.has_key?(:verify_peer)
```

Comment:
We need to fix indentation for this right

Reply:
What do you mean "fix indentation"? I just copied-and-pasted (and changed a word).

Reply:
Yeah, actually I saw it. But when I opened updated code in a separate file, it looks bit different. Feels like `Unless` is a part of the warn message.

Reply:
Sorry, I don't understand what you mean. As far as I’m aware, I haven't changed anything, and I don't really want to spend time changing the indentation of the existing code. Is there something that I’ve unintentionally changed? If not, can we resolve the conversation?

Reply:
Sure, I just wanted the code to be readable. If that part of the code makes sense to you while reading, we are good.